fwm-docs-qs/azuredeploy.json

```diff
@@ -32,95 +32,115 @@
   "resources": [
     {
       "type": "Microsoft.Network/virtualWans",
-      "apiVersion": "2019-08-01",
+      "apiVersion": "2020-06-01",
       "name": "VWan-01",
       "location": "[parameters('location')]",
       "properties": {
         "disableVpnEncryption": false,
         "allowBranchToBranchTraffic": true,
-        "allowVnetToVnetTraffic": false,
         "office365LocalBreakoutCategory": "None",
         "type": "Standard"
       }
     },
     {
       "type": "Microsoft.Network/virtualHubs",
-      "apiVersion": "2020-04-01",
+      "apiVersion": "2020-06-01",
       "name": "Hub-01",
       "location": "[parameters('location')]",
       "dependsOn": [
         "[resourceId('Microsoft.Network/virtualWans', 'VWan-01')]"
       ],
       "properties": {
-        "virtualNetworkConnections": [
-          {
-            "name": "hub-spoke",
-            "properties": {
-              "remoteVirtualNetwork": {
-                "id": "[resourceId('Microsoft.Network/virtualNetworks', 'Spoke-01')]"
-              },
-              "allowHubToRemoteVnetTransit": true,
-              "allowRemoteVnetToUseHubVnetGateways": false,
-              "enableInternetSecurity": true
-            }
-          }
-        ],
         "addressPrefix": "10.1.0.0/16",
         "virtualWan": {
           "id": "[resourceId('Microsoft.Network/virtualWans', 'VWan-01')]"
-        },
-        "azureFirewall": {
-          "id": "[resourceId('Microsoft.Network/azureFirewalls', 'AzfwTest')]"
         }
       }
     },
+    {
+      "type": "Microsoft.Network/virtualHubs/hubVirtualNetworkConnections",
+      "apiVersion": "2020-06-01",
+      "name": "Hub-01/hub-spoke",
+      "properties": {
+        "remoteVirtualNetwork": {
+          "id": "[resourceId('Microsoft.Network/virtualNetworks', 'Spoke-01')]"
+        },
+        "allowHubToRemoteVnetTransit": true,
+        "allowRemoteVnetToUseHubVnetGateways": false,
+        "enableInternetSecurity": true,
+        "routingConfiguration": {
+          "associatedRouteTable": {
+            "id": "[resourceId('Microsoft.Network/virtualHubs/hubRouteTables', 'Hub-01', 'RT_VNet')]"
+          },
+          "propagatedRouteTables": {
+            "labels": [
+              "VNet"
+            ],
+            "ids": [
+              {
+                "id": "[resourceId('Microsoft.Network/virtualHubs/hubRouteTables', 'Hub-01', 'RT_VNet')]"
+              }
+            ]
+          }
+        }
+      },
+      "dependsOn": [
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

i.e. move this above properties

Reply:
Moved depensOn above properties and pushed a new commit